### PR TITLE
javascript-forms - Michael Moncera

### PR DESCRIPTION
**Pull request by mmoncera**
https://user-images.githubusercontent.com/67080126/141904143-075c3590-be5d-4a11-bc2f-a66cd0a7d682.mov

https://user-images.githubusercontent.com/67080126/141904251-2e875446-eeb3-4d79-a863-f444e697becb.mov

### CODE REVIEW

File: javascript-forms/form-values.js [hunk 1]
```diff
@@ -0,0 +1,13 @@
+var $forms = document.querySelector('#contact-form');
+
+$forms.addEventListener('submit', function (event) {
+  event.preventDefault();
+
+  console.log('messageData:', {
```
Comment (Zhiwutian) [inline javascript-forms/form-values.js:6]: make sure you are actually creating the object then logging it out.

**Review by Zhiwutian — CHANGES_REQUESTED** [review]
small update!

File: javascript-forms/form-values.js (hunk 1 above)
Comment (mmoncera) [reply javascript-forms/form-values.js]: https://user-images.githubusercontent.com/67080126/141912518-296a4e7d-a1b1-4c9a-8773-3b093af8c832.mov

**Review by Zhiwutian — APPROVED** [review]
👍🏼

**Pull request merged**